content/github/setting-up-and-managing-your-github-user-account/types-of-emails-github-sends.md

```diff
@@ -25,6 +25,17 @@ You can also choose which type of email updates you'd like to receive on convers
 
 If you've upgraded to paid products or features, then you'll receive billing receipts at the account's primary email address. For more information, see "[Setting your billing email](/articles/setting-your-billing-email)."
 
+### Customer research emails
+
+{% data variables.product.product_name %} occasionally seeks customers to participate in research sessions to help us build a better GitHub. These are conducted remotely, open to customers worldwide, and may include:
```

Comment:
Do we want to use the term "customers" in the description here? While "customer research" is a clearly defined term, I wonder if it might be better use a more neutral term in the description. Since you can sign up for a free account, I'm not sure that all users would consider themselves customers.